### DISCUSSION TITLE
component-gap guideline suggestion

### DISCUSSION THREAD

**Discussion by Heulitig** [fastn-stack/fastn#845] [Ideas & RFCs]
Instead of specifying the exact number of lines of spacing between any two components we should give it a range like 
5-10 lines of space instead of giving exact 10 lines of space. The users will force to count lines before giving spacing between components which doesnt seem feasible at all times while writing any fastn code.

[component-gap guideline](https://fastn.io/formatting/#component-gap)

**Comment by amitu** [comment]
One thing to note: once we have [`fastn fmt`](https://github.com/orgs/fastn-stack/discussions/723), the users would not have to count.
